tests/show_techsupport/test_techsupport.py

```diff
@@ -0,0 +1,240 @@
+        'acl_table_name':  name,
+    }
+
+    logger.info('extra variables for ACL table:\n{}'.format(pprint.pformat(extra_vars)))
+    duthost.host.options['variable_manager'].extra_vars.update(extra_vars)
+
+    duthost.template(src=os.path.join(TEMPLATE_DIR, ACL_RULES_FULL_TEMPLATE),
+                    dest=dut_conf_file_path)
+
+    logger.info('applying {}'.format(dut_conf_file_path))
+    duthost.command('config acl update full {}'.format(dut_conf_file_path))       
+
+
+@pytest.fixture(scope='function')
+def acl_setup(duthost):
+    """
+    setup fixture gathers all test required information from DUT facts and testbed
+    :param duthost: DUT host object
+    :return: dictionary with all test required information
+    """  
+
+    logger.info('creating temporary folder for test {}'.format(DUT_TMP_DIR))
+    duthost.command("mkdir -p {}".format(DUT_TMP_DIR))
+
+    setup_information = {
+        'dut_tmp_dir': DUT_TMP_DIR,
+    }
+
+    logger.info('setup variables {}'.format(pprint.pformat(setup_information)))
+
+    yield setup_information
+
+    logger.info('removing {}'.format(DUT_TMP_DIR))
+    duthost.command('rm -rf {}'.format(DUT_TMP_DIR))
+
+
+def teardown_acl(dut):
+    """
+    teardown ACL rules after test by applying empty configuration
+    :param dut: DUT host object
+    :param setup: setup information
+    :return:
+    """
+    dst = DUT_TMP_DIR
+    logger.info('removing all ACL rules')
+    # copy rules remove configuration
+    dut.copy(src=os.path.join(FILES_DIR, ACL_REMOVE_RULES_FILE), dest=dst)
+    remove_rules_dut_path = os.path.join(dst, ACL_REMOVE_RULES_FILE)
+    # remove rules
+    logger.info('applying {}'.format(remove_rules_dut_path))
+    dut.command('config acl update full {}'.format(remove_rules_dut_path))
+
+
+@pytest.fixture(scope='function')
+def acl(duthost, acl_setup):
+    """
+    setup/teardown ACL rules based on test class requirements
+    :param duthost: DUT host object
+    :param acl_setup: setup information
+    :return:
+    """
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix='acl')
+    loganalyzer.load_common_config()
+
+    try:
+        loganalyzer.expect_regex = [LOG_EXPECT_ACL_RULE_CREATE_RE]
+        with loganalyzer:
+            setup_acl_rules(duthost, acl_setup)
+    except LogAnalyzerError as err:
+        # cleanup config DB in case of log analysis error
+        teardown_acl(duthost)
+        raise err    
+
+    try:
+        yield
+    finally:
+        loganalyzer.expect_regex = [LOG_EXPECT_ACL_RULE_REMOVE_RE]
+        with loganalyzer:
+            teardown_acl(duthost)
+
+
+
+
+#########################################
+########### MIRRORING PART ##############
+#########################################
+
+
+MIRROR_RUN_DIR = os.path.join('mirror_tmp', os.path.basename(BASE_DIR))
+EVERFLOW_TABLE_NAME = "EVERFLOW"
+SESSION_NAME = "test_session_1"
+session_info = {
+    'name' : SESSION_NAME,
+    'src_ip' : "1.1.1.1",
+    'dst_ip' : "2.2.2.2",
```

Comment:
Mirror session will not become active if '2.2.2.2' cannot be reached, e.g. no route to 2.2.2.2.
Can you get dst IP as one of neighbor's IP from minigraph facts? Note, this will not work for ptf32, I think you can skip this test if there are no neighbors